frontend/src/components/worksheets/NewUpload/NewUpload.jsx

```diff
@@ -92,7 +92,10 @@ class NewUpload extends React.Component<{
             if (after_sort_key || after_sort_key === 0) {
                 url += `&after_sort_key=${after_sort_key}`;
             }
-
+            // current focused item is an image block

```

Comment:
In the comment, explain why we need to pass after_image to the backend.

Reply:
solved